### PR TITLE
New introduction into mail server settings

### PR DESCRIPTION
**Pull request by dahlberg-fkie**
Clear up introduction of mail server settings chapter:
* better distinguish between MSA, MTA and MDA
* clarify common settings, which may be used as basis for the concrete configuration examples in the following sections

### CODE REVIEW

File: src/practical_settings/mailserver.tex [hunk 1]
```diff
@@ -1,47 +1,74 @@
...
-  \item do not modify the cipher suite for client mode
+  \item accept all cipher suites, as the alternative would be to fall back to
+    cleartext transmission
 \end{itemize*}
 
 For MSA operation we recommend:
 \begin{itemize*}
-  \item listen on submission port 587
+  \item listen on submission port 587 with mandatory STARTTLS
+  \item optionally listen on port 465 with Implicit TLS
```
Comment (sebix) [inline src/practical_settings/mailserver.tex]: 465 can also be used by the MTA.

File: src/practical_settings/mailserver.tex [hunk 2]
```diff
@@ -1,47 +1,74 @@
...
   \item enforce SMTP AUTH even for local networks
-  \item do not allow SMTP AUTH on unencrypted connections
-  \item optionally use the recommended cipher suites if (and only if) all your connecting MUAs support them
+  \item make sure that SMTP AUTH is not allowed on unencrypted connections
```
Comment (sebix) [inline src/practical_settings/mailserver.tex]: `make sure that` could be substituted by `ensure` for a better style

**Comment by sebix** [comment]
Thanks for your work!

+1 on my part

**Comment by 2001db8** [comment]
Looks good for me too.

**Comment by azet** [comment]
Merged. Thanks for your contribution.

**Pull request merged**